packages/golden-test-generator/src/index.ts

```diff
@@ -70,47 +70,67 @@ program
       const fileName = path.join(outDir, `address-balances-${hash(content)}.json`);
 
       logger.info(`Writing ${fileName}`);
-      await writeFile(fileName, JSONBig.stringify(content, undefined, 2));
+      await writeFile(fileName, JSON.stringify(toSerializableObject(content), undefined, 2));
       process.exit(0);
     } catch (error) {
       console.error(error);
       process.exit(1);
     }
   });
 
-program
-  .command('get-block')
-  .description('Dump the requested blocks in their raw structure')
-  .argument('[blockHeights]', 'Comma-separated list of blocks by number', (blockHeights) =>
+const mapBlockHeights = (blockHeights: string) => 
     blockHeights
       .split(',')
       .filter((b) => b !== '')
-      .map((blockHeight) => Number.parseInt(blockHeight))
-  )
+      .flatMap((blockHeightSpec) => {
+        const [from, to] = blockHeightSpec.split('..').map(blockHeight => Number.parseInt(blockHeight));
+        if (!to) { // 0 is not supported, as such range doesn't make sense
+          if (!Number.isNaN(from)) return [from]; // single block
+          throw new Error('blockHeights must be either numbers or ranges, see --help')
+        }
+        const result: number[] = [];
+        for (let blockHeight = from; blockHeight <= to; blockHeight++) {
+          result.push(blockHeight)
+        }
+        return result;
+      });
+
+program
+  .command('chain-sync')
```

Comment:
I'm not sure this is a better name for the command as it's not describing the outcome, and chain-sync is just the underlying protocol. How about `project-chain-to-file` or something like that?

Reply:
How about `chain-sync-events`?

Reply:
Is the output event data or block data?

Reply:
Event data: either roll forward or roll backward

Reply:
Let's go with `chain-sync-events`

Reply:
Done in 6e460e50326939386ed249eef18ac9735d04f1a7. It will create a conflict with 42e7c3d2230fe54f5e6bee67dd5027585f35c67f when rebasing, but I think it should be ok to just squash the 3 commits into 1.